examples/preprocessing/plot_discretization_classification.py

```diff
@@ -62,32 +62,49 @@ def get_name(estimator):
 
 # list of (estimator, param_grid), where param_grid is used in GridSearchCV
```

Comment:
Here you could add a comment similar to this one:

```
    # The parameter spaces in this example are limited to a narrow band in order to speed up the example.
    # In a real use-case, you should use a broader search space for the algorithms you use.
```

Reply:
Thank you @adrinjalali , I have added a comment.